README.md

```diff
@@ -69,7 +69,7 @@ steps:
 ```yaml
 jobs:
   build:
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-20.04
```

Comment:
What do you think about switching it to latest?

Reply:
both intents are valid. for stricter reproducible build, one might choose to set the version.

let me know if I'm to update the PR

Reply:
For a readme, I think `latest` is fine.

Reply:
updated